src/elements/hero/src/index.tsx

```diff
@@ -83,47 +85,50 @@ const Hero: React.FC<Props> = ({
                 {breadcrumbWithVariant}
               </div>
             )}
+
             <div
               sx={{
                 position: 'relative' as const,
                 display: 'block',
                 flexDirection: 'row-reverse' as const
               }}
             >
-              <div
-                sx={{
-                  position: 'absolute' as const,
-                  left: [0, '45%'],
-                  right: 0,
-                  top: 0,
-                  bottom: 0,
-                  display: [
-                    fgImageOnMobile ? 'block' : 'none',
-                    fgImageOnTablet ? 'block' : 'none',
-                    'block'
-                  ],
-                  ...(fgImage && fgImageType === 'background'
-                    ? {
-                        backgroundImage: `url(${fgImage})`,
-                        backgroundRepeat: 'no-repeat',
+              {!advertisment ? (
```

Comment:
Can you use && for better readability? The `null` after the `:` is unnecessary.

Reply:
Done